### ISSUE TITLE
MarkdownField Issues Again

### ISSUE THREAD

**Issue by Felli** [humhub/humhub#2697]
So it would seem that MarkdownField.php has broken again? If I use the Markdown option for Profile attributes it breaks `/user/account/edit`

HumHub v1.2.2

```
yii\base\UnknownPropertyException
yii\base\UnknownPropertyException: Setting unknown property: humhub\widgets\MarkdownField::readOnly in /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/Component.php:201 Stack trace:
#0 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/BaseYii.php(529): yii\base\Component->__set('readOnly', true)
#1 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/Object.php(105): yii\BaseYii::configure(Object(humhub\widgets\MarkdownField), Array)
#2 [internal function]: yii\base\Object->__construct(Array)
#3 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/di/Container.php(381): ReflectionClass->newInstanceArgs(Array)
#4 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/di/Container.php(156): yii\di\Container->build('humhub\\widgets\\...', Array, Array)
#5 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/BaseYii.php(348): yii\di\Container->get('humhub\\widgets\\...', Array, Array)
#6 /home/USER/HUMHUB_SITE/protected/humhub/components/Widget.php(55): yii\BaseYii::createObject(Array)
#7 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/widgets/ActiveField.php(725): humhub\components\Widget::widget(Array)
#8 /home/USER/HUMHUB_SITE/protected/humhub/compat/HForm.php(294): yii\widgets\ActiveField->widget('humhub\\widgets\\...', Array)
#9 /home/USER/HUMHUB_SITE/protected/humhub/compat/HForm.php(140): humhub\compat\HForm->renderField('verified', Array, Array)
#10 /home/USER/HUMHUB_SITE/protected/humhub/compat/HForm.php(136): humhub\compat\HForm->renderElements(Array, Array)
#11 /home/USER/HUMHUB_SITE/protected/humhub/compat/HForm.php(122): humhub\compat\HForm->renderElements(Array)
#12 /home/USER/HUMHUB_SITE/protected/humhub/modules/user/views/account/edit.php(10): humhub\compat\HForm->render(Object(yii\bootstrap\ActiveForm))
#13 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/View.php(330): require('/home/USER/...')
#14 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/View.php(250): yii\base\View->renderPhpFile('/home/USER/...', Array)
#15 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/View.php(152): yii\base\View->renderFile('/home/USER/...', Array, Object(humhub\modules\user\controllers\AccountController))
#16 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/Controller.php(381): yii\base\View->render('edit', Array, Object(humhub\modules\user\controllers\AccountController))
#17 /home/USER/HUMHUB_SITE/protected/humhub/modules/user/controllers/AccountController.php(85): yii\base\Controller->render('edit', Array)
#18 [internal function]: humhub\modules\user\controllers\AccountController->actionEdit()
#19 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#20 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/Controller.php(156): yii\base\InlineAction->runWithParams(Array)
#21 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/Module.php(523): yii\base\Controller->runAction('edit', Array)
#22 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/web/Application.php(102): yii\base\Module->runAction('user/account/ed...', Array)
#23 /home/USER/HUMHUB_SITE/protected/vendor/yiisoft/yii2/base/Application.php(380): yii\web\Application->handleRequest(Object(humhub\components\Request))
#24 /home/USER/HUMHUB_SITE/index.php(25): yii\base\Application->run()
#25 {main}
```

**Comment by githubjeka** [comment]
Confirm

**Comment by githubjeka** [comment]
Oh no, after logout , login by normal(not LDAP) user, logout, and login by LDAP user - profile/edit work fine 
😕

**Comment by buddh4** [comment]
does this only occure for readonly fields?

**Comment by Felli** [comment]
It only happened when a Markdown field attribute was applied.

**Comment by BastusIII** [comment]
There was an issue with the used case for the readonly attribute in MarkdownField.php, `readonly` instead of `readOnly` This should be solved now.

**Comment by Felli** [comment]
It's fixed with bb75a888caef8b3a539f5002e9a27560e02b3ef8
